eng/scripts/dotnet-install.ps1

```diff
@@ -0,0 +1,976 @@
+#
```

Comment:
Do we need to check-in this entire script? Or should we use the well known version, similar to what the .NET team does, see https://github.com/dotnet/runtime/blob/master/eng/common/tools.ps1#L193 as an example. 

It might also be better to read the version from the global.json file similar to what they do in as well https://github.com/dotnet/runtime/blob/master/eng/common/tools.ps1#L140.

Reply:
Ahh, that's nice! Thanks for the suggestion.